app/controllers/posts_controller.rb

```diff
@@ -1,16 +1,4 @@
 class PostsController < ApplicationController
-  def index
-    @user = User.includes(:posts).find(params[:user_id])
-    @posts = @user.posts.includes(:comments)
-  end
-
-  def show
-    @user = User.includes(posts: :comments).find(params[:user_id])
-    @post = Post.find(params[:id])
-    @posts = @user.posts
-  end
-
-  load_and_authorize_resource
   def index
     @user = User.find(params[:user_id])
     @posts = Post.includes(:author, :comments).where(author_id: @user)
```

Comment:
- [ ] @Team Great work on your project, however it doesn't solve all the N + 1 problems, your previous code  `@posts = @user.posts.includes(:comments)` did solve this problem  as it fetched all posts and their comments for a user, you can see how to integrate it in your new code, also you forgot to  include the screenshots of your app's logs before and after the fix